src/lib/Libutil/misc_utils.c

```diff
@@ -75,6 +75,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <pwd.h>
+#include <malloc.h>
```

Comment:
Should this only be included when HAVE_MALLOC_INFO is defined?

Reply:
I've addressed your comment. Please check.